orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/kato/pipeline/support/ResizeStrategy.groovy

```diff
@@ -56,6 +56,13 @@ interface ResizeStrategy {
     Integer max
     Integer desired
     Integer min
+
+    // from https://codereview.stackexchange.com/questions/8801/returning-groovy-class-fields-as-a-map
+    public Map asMap() {
```

Comment:
I think you can just call `.properties()` to get the value as a map?

Reply:
or I guess that has some groovy magic-junk in it?

Reply:
I get this though

```
MissingMethodException: No signature of method: Capacity.properties() is applicable for argument types: () values: []
```

Reply:
If you're still interested, it's a property, not a method:
```
> import com.netflix.spinnaker.orca.kato.pipeline.support.ResizeStrategy.Capacity
> 
> Capacity c = new Capacity(3,2,1)
> print(c.properties)
[class:class com.netflix.spinnaker.orca.kato.pipeline.support.ResizeStrategy$Capacity, max:3, min:1, desired:2]
```